Library/Formula/denominator.rb

```diff
@@ -1,11 +1,12 @@
 class Denominator < Formula
   # This is a shell script wrapping a java jar; download as single file using nounzip
-  homepage "https://github.com/Netflix/denominator/tree/master/cli"
-  url "http://dl.bintray.com/content/netflixoss/denominator/denominator-cli/release/4.3.3/denominator?direct",
+  homepage "https://github.com/Netflix/denominator/tree/v4.4.0/cli"
+  url "https://bintray.com/artifact/download/netflixoss/maven/com/netflix/denominator/denominator-cli/4.4.0/denominator-cli-4.4.0-fat.jar",
     :using  => :nounzip
```

Comment:
We auto-select `nounzip` for .jar URLs, so I think this `using` can be removed (but please test, thanks.)